components/script/stylesheet_loader.rs

```diff
@@ -92,6 +96,35 @@ impl FetchResponseListener for StylesheetContext {
 
     fn process_response(&mut self,
                         metadata: Result<FetchMetadata, NetworkError>) {
+        let stylesheet: Option<Root<CSSStyleSheet>>;
+        let elem = self.elem.root();
+        if let Some(element) = elem.downcast::<HTMLLinkElement>() {
+            stylesheet = element.get_cssom_stylesheet();
+        } else if let Some(element) = elem.downcast::<HTMLMetaElement>() {
+            stylesheet = element.get_cssom_stylesheet();
```

Comment:
I don't believe stylesheets originating from meta elements can ever cause a network load to start, so this shouldn't be necessary here.